core/src/main/java/tc/oc/pgm/stats/StatsMatchModule.java

```diff
@@ -12,16 +10,9 @@
 import com.google.common.collect.Table;
 import com.google.common.collect.Tables;
 import java.time.Duration;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.UUID;
+import java.util.*;
```

Comment:
avoid wildcard imports, configure your IDE not to use them

Reply:
Sure